chain/src/CapTable.sol

```diff
@@ -290,4 +296,35 @@ contract CapTable is Ownable {
     function _safeNow() internal view returns (uint40) {
         return uint40(block.timestamp);
     }
+
+    /* RBAC */
+
+    modifier isOperator() {
+        // TODO: should admins be operators?
+        require(operators.has(msg.sender) || admins.has(msg.sender), "Does not have operator role");
+        _;
+    }
+
+    modifier isAdmin() {
+        require(admins.has(msg.sender), "Does not have admin role");
+        _;
+    }
+
+    // TODO: kent -- is there a more elegant way for the below? I hate boilerplate code bloat
+    function addAdmin(address memory addr) external isAdmin {
+        admins.add(addr);
```

Comment:
I can't think of another way top of mind but it definitely adds code bloat.
Should we create a Roles internal library that contains these functions?

Reply:
as discussed, we are going to keep as is